comp/core/workloadmeta/impl/dump_test.go

```diff
@@ -190,6 +194,8 @@ Health:
 Created At: 0001-01-01 00:00:00 +0000 UTC
 Started At: 0001-01-01 00:00:00 +0000 UTC
 Finished At: 0001-01-01 00:00:00 +0000 UTC
+----------- SBOM -----------
+SBOM is nil
```

Comment:
Should we exercise a test in which the SBOM is not nil?

Reply:
Added test